substrate/frame/src/lib.rs

```diff
@@ -32,10 +32,17 @@
 //!
 //! ## Usage
 //!
-//! The main intended use of this crate is for it to be imported with its preludes:
+//! This crate is organized into 3 stages:
+//!
+//! 1. preludes: [`prelude`], [`testing_prelude`] and [`runtime::prelude`], [`benchmarking`],
```

Comment:
Can you add a note here about the fact that some preludes already include one another: 

The order is: 

`testing` brings in `frame::prelude::*` and `runtime::prelude`
`runtime` brings in `frame::predlude::*` 
`benchmarking` brings in `frame::predlude::*` 
 etc etc

Reply:
Added in https://github.com/paritytech/polkadot-sdk/pull/5995/commits/dc2973d0074a6a41f840b042cb204f310cb9a04d